lib/codeowners/cli/wizards/new_owner_wizard.rb

```diff
@@ -25,24 +35,46 @@ def suggest_fixing(line, new_owner)
 
         private
 
-        def prompt(line, new_owner)
-          ask(<<~QUESTION, limited_to: %w[y r i q])
-            Unknown owner: #{new_owner} for pattern: #{line.pattern}. Add owner to the OWNERS file?
-            (y) yes
-            (r) rename owner
-            (i) ignore owner in this session
-            (q) quit and save
+        def suggest_name_from_owners_list(new_owner)
+          require 'fuzzy_match'
+          search = FuzzyMatch.new(@owners_list.owners)
+          (suggested_owner, dice, _lev) = search.find_with_score(new_owner)
+          return suggested_owner if dice && dice > 0.6
+        end
+
+        def prompt(line, new_owner, suggested_owner)
+          prompt_options = build_prompt_options(suggested_owner)
+          ask(<<~QUESTION, limited_to: prompt_options.keys)
+            #{question_body(line, new_owner, suggested_owner)}
+            #{question_options(prompt_options)}
           QUESTION
         end
 
+        def question_body(line, new_owner, suggested_owner)
+          prompt = "Unknown owner: #{new_owner} for pattern: #{line.pattern}."
+          if suggested_owner
+            prompt + " Did you mean #{suggested_owner}?"
+          else
+            prompt + ' Add owner to the OWNERS file?'
```

Comment:
Maybe we can say "Choose an option" since `Add owner` is only one of the options.

Reply:
done :heavy_check_mark: